pkg/envoy/proxy_test.go

```diff
@@ -63,7 +64,8 @@ var _ = Describe("Test proxy methods", func() {
 
 			firstNonce := proxy.GetLastSentNonce(TypeCDS)
 			Expect(firstNonce).ToNot(Equal(uint64(0)))
-
+			// Accomodate platforms where time now has lower accuracy.
```

Comment:
Would make this comment more specific to name the platform so readers have context.
`// XXX(Windows): accommodate for slowness where time has lower accuracy...`

Reply:
WDYT `// Platform(<name>) as a a comment pattern for platform specific issues. I can add any windows specific comments there myself and it might be useful in the future for `arm` or specific quirks in POSIX OS

Reply:
I'd also suggest to put this under a conditional so we don't mask bugs arising from timing issues:
```
if runtime.GOOS == "windows" {
    time.Sleep(x)
}
```